greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-5343: Plastics theme: items should be center aligned

Ref: https://jira.greenpeace.org/browse/PLANET-5343
Depends on https://github.com/greenpeace/planet4-base-fork/pull/89 for tests
Replaces #340

> Task column block copy, icon and button copy should be centre aligned and not left aligned

This PR aligns icon and button of plastic campaign columns to the center.  
It also changes the columns_tasks markup to be closer to columns and use flexbox by removing most of the container divs and using `block-style-image` rules.

## Tests

- Export campaigns from the [staging site](https://release.k8s.p4.greenpeace.org/international/) (Tools > Export)
- Import the file locally (Tools > Import)
- Display the campaign _Lobby for Single-Use Ban in Your Community_
- Icons, title and buttons should be centered

<img width="984" alt="Screenshot 2020-07-28 at 14 45 45" src="https://user-images.githubusercontent.com/617346/88666991-1793c280-d0e1-11ea-9258-dce6ec3527d2.png">
